### PR TITLE
  Create-TF-1639-User-can-manage-label

### PR DESCRIPTION
**Pull request by dieptran88**
- View label
- Create label
- Edit 
- Delete label

### CODE REVIEW

**Comment by dieptran88** [comment]
@hoangdat @chibenwa Please review

File: docs/stories/EPIC-Label/TF-1639-user-can-manage-labels.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Name of label is displayed but I cannot edit this name
+- I can only select another color for this label then click button Save
+- The new color is update for the label and all the messages that have this label will be updated with new color too.
+
+![image](https://user-images.githubusercontent.com/68209176/226866342-6be723c5-c096-4b0e-905e-8517c77eee2d.png)
+
+![image](https://user-images.githubusercontent.com/68209176/226866412-3d5aca75-e290-4bb4-a8b5-676577571ef0.png)
+
+
+#### UC4. Delete a label
```
Comment (chibenwa) [inline docs/stories/EPIC-Label/TF-1639-user-can-manage-labels.md:50]: Deletion is hard as you need to untag all emails.

Imo drop from mvp

Comment (dieptran88) [reply docs/stories/EPIC-Label/TF-1639-user-can-manage-labels.md:50]: @hoangdat WDYT ?

Comment (hoangdat) [reply docs/stories/EPIC-Label/TF-1639-user-can-manage-labels.md:50]: please remove edit/delete use-case.

**Comment by dieptran88** [comment]
@hoangdat please help me to review and merge

**Review by hoangdat — APPROVED** [review]

**Pull request merged**